### DISCUSSION TITLE
Battery/Power notifications via mako

### DISCUSSION THREAD

**Discussion by nnuel** [Manjaro-Sway/manjaro-sway#232] [Ideas]
I came across this handy tool: https://sr.ht/~kennylevinsen/poweralertd/

With the following setting in the sway-config, it alerts you when the powercord is plugged in or accidentally yanked loose. 
Not sure if it falls under user preference and how it interacts with bluetooth mice/keyboards, but will test that later.

```
exec poweralertd -s -i "line power"
```

The `-s` flag ignores the notification at boot or startup, and the `-i` flag makes it so that it doesn't show the AC power line info and the battery charging/discharging.

**Comment by boredland** [comment]
That is prettty cool. We should probably package poweralertd and add it as an extra dependency.

**Comment by boredland** [comment]
poweralertd is packaged and enabled by default!
